frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/keys_pressed.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class KeysPressed extends StatefulWidget {
+  final Widget child;
+
+  const KeysPressed({
+    super.key,
+    required this.child,
+  });
+
+  @override
+  State<KeysPressed> createState() => _KeysPressedState();
+}
+
+class _KeysPressedState extends State<KeysPressed> {
+  final _focusNode = FocusNode();
+
+  bool _isAltPressed = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return KeyboardListener(
+      focusNode: _focusNode,
+      onKeyEvent: (event) {
```

Comment:
If the user releases the Alt key, how does the `isAltPressed` value get reset back to false?